### DISCUSSION TITLE
Nwipe & ShredOS with PDF Erasure reports

### DISCUSSION THREAD

**Discussion by PartialVolume** [AndreRenaud/PDFGen#130] [Show and tell]
Hi Andre, I just wanted to compliment you on the work you have done on this software. Very nice work. I've just been using it to implement a new feature in [nwipe (disk erasure)](https://github.com/martijnvanbrummelen/nwipe/) and [ShredOS (bootable disk erasure )](https://github.com/PartialVolume/shredos.x86_64)

 [Thread containing 1st draft of PDF](https://github.com/martijnvanbrummelen/nwipe/issues/426#issuecomment-1427138475)

Thanks again.

**Comment by AndreRenaud** [comment]
Thanks for letting me know - it's always nice to know the code is useful to someone. Let me know if you have any ideas for things that might make your life easier. The thing you're producing (test reports) is very similar to the original reason why I wrote the library. We used it to produce the barcodes & output from manufacturing production systems for traceability.

**Reply by PartialVolume** [reply]
>Let me know if you have any ideas for things that might make your life easier. 

Looks pretty much perfect for what I need, however there was one thing related to the colour of the text.

I think there are only a limited number of definitions in the header for PDF_BLACK, PDF_WHITE, PDF_RED, PDF_GREEN and PDF_BLUE. Is there some technical reason for there not being more or can I just create a few more? If I did create a few more colours, like YELLOW, PURPLE, BROWN etc are you ok with me doing a PR on your code?

**Reply by AndreRenaud** [reply]
There's no problem having more of them. If there were more, I'd probably change it to an enum instead of #defines, but whatever is simplest for you is fine. A PR would be great - thanks.
